lumicks/pylake/nb_widgets/tests/test_kymotracker_widget.py

```diff
@@ -123,16 +134,23 @@ def test_stitch(kymograph, mockevent):
 
     assert len(kymo_widget.lines) == 2
 
+    def kymo_units(time_idx, coord_idx):
+        return kymograph.line_time_seconds * time_idx, kymograph.pixelsize_um[0] * coord_idx
```

Comment:
I find this confusing since `kymo_units` in previous test cases had a different order of parameters. Unfortunately, I don't have good suggestions...

Maybe rename it to something like `kymo_time_position`? Or break it into 2 functions and use those (`in_s` and `in_um`).

Reply:
Good point! Decided to go with the following:

`in_um, in_s = calibrate_to_kymo(kymograph)`

And then using those.